### ISSUE TITLE
Rendering checkboxes in HTML

### ISSUE THREAD

**Issue by tamasgal** [JuliaDocs/Documenter.jl#2093]
I just noticed that checkboxes are not rendered in HTML, so

```
- [ ] foo
- [ ] bar
```

Will be displayed as

- [] foo
- [] bar

I cannot work on this at the moment, but maybe someone knows how to fix this quickly.

**Comment by mortenpi** [comment]
That's not standard markdown, and the Julia Markdown parser doesn't support this. I don't think CommonMark.jl supports it either.. @MichaelHatherly? So first step would be to get parser-level support for this.

**Comment by MichaelHatherly** [comment]
Yeah, unsupported by `CommonMark.jl` as well. Would be a good addition as an extension.
